loading.bs

```diff
@@ -331,6 +338,222 @@ Note: Applying the signed exchange's response here has the effect of letting a
 newer HTTP cache entry override a signed exchange's content, and of not storing
 the signed exchange's response in the HTTP cache.
 
+# Subresource substitution # {#subresource-substitution}
+
+To let the UA know that subresources of a page it's prenavigating to, can
+be retrieved as signed exchanges from the source site, this section introduces
+an extension to the HTTP Link header.
+
+* A <dfn dfn-type=dfn>allowed-alt-sxg link</dfn> is a relationship that is used
+    in HTTP <a http-header>Link</a> header of inner HTTP response of main
+    resource signed exchange to indicate that the subresource of the
+    [=link target=] can be loaded from different URL as a signed exchange. This
+    link has a
+    [rel](https://html.spec.whatwg.org/multipage/semantics.html#attr-link-rel)
+    parameter contains `allowed-alt-sxg` and a `header-integrity` parameter set
+    to the value of the [=header integrity value=] of the [=alternate signed
+    exchange=] 
+* A <dfn dfn-type=dfn>alternate signed exchange link</dfn> is a relationship
```

Comment:
This is not actually a link relationship since it's not a keyword. Imitating https://www.w3.org/TR/resource-hints/#dfn-resource-hint-link, how about:

```suggestion
* An <dfn>alternate signed exchange link</dfn> is a `` `Link` `` header sent with a signed exchange |S|, with the [=alternate=] link type, the `type` parameter set to `application/signed-exchange`, and a [=Link Context=]. The [=Link Context=] MUST be the [=Link Target=] of a [=preload=] `` `Link` `` header inside the signed content of |S|, and the [=alternate signed exchange link=] means that this resource can also be found inside the signed exchange at the [=Link Target=] of the [=alternate signed exchange link=].

    <div class="example" id="example-alternate-signed-exchange-link">
    ```http
    Link: <...>; ...
    ```
  </div>
```

Add the Link Target (https://tools.ietf.org/html/rfc8288#section-3.1) and Link Context (https://tools.ietf.org/html/rfc8288#section-3.2) definitions to the `<pre class='anchors'>` so they appear correctly in the References section.

You may have to fiddle with the bikeshed markdown to get it to link to https://html.spec.whatwg.org/multipage/links.html#rel-alternate and look right. See https://tabatkins.github.io/bikeshed/#autolinking.

This would probably make more sense with a diagram showing where in the inner and outer signed exchanges the headers appear. Try to make one with https://plantuml.com/component-diagram#58bb821cf05c48a2?

Reply:
> You may have to fiddle with the bikeshed markdown to get it to link to https://html.spec.whatwg.org/multipage/links.html#rel-alternate and look right. See https://tabatkins.github.io/bikeshed/#autolinking.

Sorry I may not understand well how bikeshed works.
I created a PR to the HTML spec to export `rel` and `alternate`. https://github.com/whatwg/html/pull/5843
Do I need to wait for the PR to be merged?